plugins/tech-radar/src/components/RadarComponent.tsx

```diff
@@ -68,7 +68,7 @@ const RadarComponent = (props: TechRadarComponentProps): JSX.Element => {
           };
         }),
         moved: entry.timeline[0].moved,
-        description: entry.timeline[0].description,
+        description: entry.description || entry.timeline[0].description,
```

Comment:
I've maintained backward compatibility here, though not sure if it is 100% the correct path.